multirun/deps.bzl

```diff
@@ -4,7 +4,7 @@ def multirun_dependencies():
     _maybe(
         http_archive,
         name = "bazel_skylib",
-        sha256 = "2ef429f5d7ce7111263289644d233707dba35e39696377ebab8b0bc701f7818e",
+        sha256 = "2ea8a5ed2b448baf4a6855d3ce049c4c452a6470b1efd1504fdb7c1c134d220a",
```

Comment:
```bash
curl --output ./1.tar.gz https://codeload.github.com/bazelbuild/bazel-skylib/tar.gz/0.8.0
shasum -a 256 ./1.tar.gz
2ea8a5ed2b448baf4a6855d3ce049c4c452a6470b1efd1504fdb7c1c134d220a  ./1.tar.gz
```

Reply:
This is because `https://github.com/*/*/archive/*/*.tar.gz` is generated dynamically by GitHub and they have probably changed the software that constructs this file and hence it does not match anymore. The right thing here is to either start using [git_repository](https://docs.bazel.build/versions/master/repo/git.html#git_repository) or use a released artifact from https://github.com/bazelbuild/bazel-skylib/releases/tag/0.8.0. Just updating the hash is a temporary measure. I'll merge this PR to unblock you and then I'll fix it properly. Cheers.

Reply:
thanks ash, it works well now